File: Frontend/UserRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Frontend
{
    class UserRequest
    {
        public const string LOGIN = "LOGIN";
        public const string REGISTER = "REGISTER";
        public const string CHECKIN = "CHECKIN";
        public const string LOGOUT = "LOGOUT";
        public const string RETRIEVE = "RETRIEVE";
        public const string RETRIEVEALL = "RETRIEVEALL";
        public const string DELETE = "DELETE";
        public const string FORGOT = "FORGOT";


        public string method { get; set; }
        public string id { get; set; }
        public Account account { get; set; }
    }
}

File: app/src/main/java/com/example/lightanalysis/models/UserRequest.java
package com.example.lightanalysis.models;

public class UserRequest {


    public String LOGIN = "LOGIN";
    public String REGISTER = "REGISTER";
    public String CHECKIN = "CHECKIN";
    public String LOGOUT = "LOGOUT";
    public String RETRIEVE = "RETRIEVE";
    public String RETRIEVEALL = "RETRIEVEALL";
    public String DELETE = "DELETE";

    public String method;
    public String id;
    public Account account;

    public String getMethod() {
        return method;
    }

    public void setMethod(String method) {
        this.method = method;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public Account getAccount() {
        return account;
    }

    public void setAccount(Account account) {
        this.account = account;
    }


}
